loki.md: Add example of Loki data source config (#29976)
grafana/grafana: docs/sources/datasources/loki.md

          name: TraceID
          url: "http://localhost:16686/trace/$${__value.raw}"
```

> Here's example of Jaeger datasource corresponding to the above example. Note Jaeger's `uid` value does match with Loki's `datasourceUid` value.

Reviewer: ```suggestion
Here's an example of a Jaeger data source corresponding to the above example. Note that the Jaeger `uid` value does match the Loki `datasourceUid` value.
```

```
datasources:
    - name: Jaeger
      type: jaeger
      url: http://jaeger-tracing-query:16686/
      access: proxy

>       # uid should be matching with datasourceUid in dervidedFields.

Reviewer: ```suggestion
      # UID should match the datasourceUid in dervidedFields.
```
